src/simulations/spatial_parameters.jl

```diff
@@ -8,6 +8,32 @@ function mean(x::AbstractArray{T}) where {T}
     sum(x) / length(x)
 end
 
+"""
+    use_lowres_clm(mesh)
+
+Returns true if the mesh is closer in resolution to 0.9x1.25 degree lat/long than 0.125x0.125 degree lat/long.
+This is used to determine which resolution of CLM data to use.
+"""
+function use_lowres_clm(mesh::ClimaCore.Meshes.AbstractCubedSphere)
+    sphere_radius = mesh.domain.radius
+    horizontal_length_scale(lat_res, long_res) =
+        sqrt(4 * pi * sphere_radius^2 / ((360 / long_res) * (180 / lat_res)))
+    highres_scale = horizontal_length_scale(0.125, 0.125)
+    lowres_scale = horizontal_length_scale(0.9, 1.25)
+    # scale in meters
+    element_scale = ClimaCore.Meshes.element_horizontal_length_scale(mesh)
+    return abs(lowres_scale - element_scale) <
+           abs(highres_scale - element_scale)
```

Comment:
I think we want to check the distance between nodes, not elements. So you might want to use `node_horizontal_length_scale` instead